.eslintrc.js

```diff
@@ -66,6 +66,7 @@ module.exports = {
       {
         patterns: [
           '@novu/shared/*',
+          '!@novu/shared/utils',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** this excludes the utils path from bad imports rule right? ‏

Reply:
Yes.